modules/controller/lib/validators/init/model_method.js

```diff
@@ -0,0 +1,15 @@
+const _ = require('lodash');
+
+module.exports = function(opts, source) {
+
+  var controllerMethod = opts.controllerMethod;
+  var modelMethod = opts.modelMethod;
+
+  // The method is on prototype for update/destroy and on the model for
+  // create: ¯\_(ツ)_/¯
+  var modelMethodFn = source.model.prototype[modelMethod] || source.model[modelMethod];
```

Comment:
stuff on the prototype is for instances (update/destroy), create is a static method on the model because you don't have an instance yet.